specification/properties/SystemMessage.json

```diff
@@ -9,9 +9,13 @@
     ],
     "example": "info"
   },
-  "filter": {
+  "resource_type": {
```

Comment:
This is the resource type that this message is related to, which could indicate to the backoffice where to display this message.